app/controllers/concerns/price_group_members_controller.rb

```diff
@@ -28,7 +28,7 @@ def create
     raise NUCore::PermissionDenied unless @price_group.can_manage_price_group_members?
 
     if price_group_member.save
-      LogEvent.log(price_group_member, :create, current_user)
+      LogEvent.log(price_group_member, :create, current_user, metadata: { member_type: price_group.member_type })
```

Comment:
Looks like you're trying to access `member_type` on the wrong object

```suggestion
      LogEvent.log(price_group_member, :create, current_user, metadata: { member_type: price_group_member.member_type })
```